SPECS/privacyidea.spec

```diff
@@ -18,7 +19,7 @@ Packager:       Cornelius Kölbel <cornelius.koelbel@netknights.it>
 BuildArch:      x86_64
 AutoReqProv:	no
 
-BuildRequires: libxml2-devel, freetype-devel, python-devel, libxslt-devel, zlib-devel, openssl-devel, python-virtualenv, gcc, createrepo
```

Comment:
Are you sure, we do not need some of the depencies? (Although they might be installed by default)

Reply:
Well, i tested building the RPMs without these packages installed, it did seem to work... We removed a few dependencies that needed building some libs.

Reply:
tested and works. We really don't need the devel build dependencies anymore.